### DISCUSSION TITLE
FNIRSI GC-01: Missing firmware

### DISCUSSION THREAD

**Discussion by GeigerMuller** [Gissio/radpro#153] [Q&A] [answered]
Hi, I can't find the original firmware for FNIRSI GC-01. According to the instructions, it should be stored here: https://github.com/Gissio/radpro/blob/main/docs/devices/FNIRSI%20GC-01/firmware, but there are only older rc versions. Can someone upload it?

**Comment by GeigerMuller** [comment]
they have it on their side: https://fnirsistore.com/fnirsi-firmware-upgrade/?srsltid=AfmBOoq1OGUjZXN-H9mdt3u41-3k1PNGMEqFCekZjCoX-39qEt8eb8PC

**Reply by Gissio** [reply, marked as answer]
You are confusing a GitHub commit with a file release.

[fnirsi-gc01-apm32f103rbt6-v1.6.bin](https://github.com/Gissio/radpro/blob/main/docs/devices/FNIRSI%20GC-01/firmware/fnirsi-gc01-apm32f103rbt6-v1.6.bin) is the same as GC01_V1.6.bin from https://fnirsistore.com/fnirsi-firmware-upgrade .
